docs/quick-ingestion-guides/powerbi/configuration.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Configuration
+---
+# Configuring Your PowerBI Connector to DataHub
+
+Now that you have created a DataHub specific Azure AD app with the relevant access in [the prior step](setup.md), it's now time to set up a connection via the DataHub UI.
+
+## Configure Secrets
+
+1. Within DataHub, navigate to the **Ingestion** tab in the top, right corner of your screen
+
+<p align="center">
+  <img width="75%" alt="Navigate to the &quot;Ingestion Tab&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_ingestion_button.png"/>
+</p>
+
+:::note
+If you do not see the Ingestion tab, please contact your DataHub admin to grant you the correct permissions
+:::
+
+2. Navigate to the **Secrets** tab and click **Create new secret**
+
+<p align="center">
+   <img width="75%" alt="Secrets Tab" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_secrets_tab.png"/>
+</p>
+
+3. Create a client id secret
+
+  This will securely store your PowerBI `Application (client) ID` within DataHub
+
+   * Enter a name like `POWER_BI_CLIENT_ID` - we will use this later to refer to the `Application (client) ID`
+   * Enter the `Application (client) ID`
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="Application (client) ID" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/powerbi/powerbi-client-id-secret.png"/>
+</p>
+
+4. Create a secret to store the Azure AD Client Secret
+
+  This will securely store your client secret"
+
+   * Enter a name like `POWER_BI_CLIENT_SECRET` - we will use this later to refer to the client secret
+   * Enter the client secret
+   * Optionally add a description
+   * Click **Create**
+
+<p align="center">
+   <img width="70%" alt="Azure AD app Secret" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/powerbi/powerbi-client-secret.png"/>
+</p>
+
+## Configure Recipe
+1.  Navigate to the **Sources** tab and click **Create new source**
```

Comment:
Let's make this section a bit more concise:


Configuring Recipe

Navigate to the **Sources** tab and click **Create new source**.
Choose PowerBI.

Inside the form, provide the following fields:

a. tenant_id: The unique identifier (GUID) of the Azure Active Directory instance, found in PowerBI Portal > ? > About PowerBI. Copy the ctid.

b. client_id: Use the secret POWER_BI_CLIENT_ID with the format "${POWER_BI_CLIENT_ID}".

c. client_secret: Use the secret POWER_BI_CLIENT_SECRET with the format "${POWER_BI_CLIENT_SECRET}".

Optionally, use the workspace_id_pattern field to filter for specific workspaces.

After completing the recipe, click Next.

Reply:
Portal doesn't shows form for PowerBI